actions/views/rhs.jsx

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {getCurrentUserId} from 'mattermost-redux/selectors/entities/users';
+import {getCurrentTeamId} from 'mattermost-redux/selectors/entities/teams';
+import {makeGetMessageInHistoryItem, makeGetCommentCountForPost} from 'mattermost-redux/selectors/entities/posts';
+import {getCustomEmojisByName} from 'mattermost-redux/selectors/entities/emojis';
+
+import {
+  addReaction,
+  removeReaction,
+  addMessageIntoHistory,
+  moveHistoryIndexBack,
+  moveHistoryIndexForward
+} from 'mattermost-redux/actions/posts';
+
+import {Posts} from 'mattermost-redux/constants';
+
+import * as PostActions from 'actions/post_actions.jsx';
+import * as GlobalActions from 'actions/global_actions.jsx';
+import * as ChannelActions from 'actions/channel_actions.jsx';
+import AppDispatcher from 'dispatcher/app_dispatcher.jsx';
+import PostStore from 'stores/post_store.jsx';
+import {EmojiMap} from 'stores/emoji_store.jsx';
+
+import {makeGetCurrentUsersLatestPost} from 'selectors/rhs';
+
+import * as Utils from 'utils/utils.jsx';
+import {ActionTypes} from 'utils/constants.jsx';
+
+import {REACTION_PATTERN} from 'components/create_post.jsx';
+
+export function updateCommentDraft(rootId, draft) {
+    return (dispatch) => {
+        PostStore.storeCommentDraft(rootId, draft);
```

Comment:
This will work but PostStore is really just dispatching to [the newly](https://github.com/mattermost/mattermost-webapp/blob/master/reducers/storage.js). So in interest in reducing usage of PostStore, can you perform the dispatch directly?